authui/src/authflowv2/password-policy.ts

```diff
@@ -114,8 +117,8 @@ async function checkPasswordStrength(
 ) {
   let isViolated = false;
   const minLevel = Number(el.getAttribute("data-min-level"));
-  const { default: zxcvbn } = await import("zxcvbn");
-  const result = zxcvbn(value);
+  const z = (await import("zxcvbn")) as unknown as Zxcvbn;
```

Comment:
Or instead of directly importing zxcvbn dynamically, we should isolate the import to a new file which defined functions to be called, and dynamically import that file instead?

In this way we can also add zxcvbn to banned imports.

Reply:
Seems like lazy loading controller then, or another way would be have a deps provider declared per controller? 🤔

Reply:
I think the controller must be loaded in authflowv2.ts, so we cannot lazy load it.

My thought was to simply having a module to "isolate" some large dependencies, and ban direct import of those dependencies using eslint.